server/eclipse-project/src/main/java/us/freeandfair/corla/model/CountyDashboard.java

```diff
@@ -545,6 +557,42 @@ public Integer estimatedBallotsToAudit() {
   }
   
   /**
+   * @return a String representation of this contest.
+   */
+  @Override
+  public String toString() {
+    return "CountyDashboard [county=" + id() + "]";
+  }
+
+  /**
+   * Compare this object with another for equivalence.
+   * 
+   * @param the_other The other object.
+   * @return true if the objects are equivalent, false otherwise.
+   */
+  @Override
+  public boolean equals(final Object the_other) {
```

Comment:
Idea: couldn't we push the dynamic type checking of `Object` into a public `equals` method on a new `AbstractModel` base class, and then have that call a method provided by the implementing class which only ever compares two of the given model? A similar strategy might help make a lot of the model code easier to change and review.